.env_sample

```diff
@@ -158,10 +158,14 @@ STACK_EXCHANGE_APP_KEY=""
 GITHUB_KEY="Optional"
 GITHUB_SECRET="Optional"
 
-# For Sendgrid email
-# https://sendgrid.com/docs/for-developers/sending-email/upgrade-your-authentication-method-to-api-keys/#upgrade-to-api-keys-for-your-smtp-integration
-SENDGRID_API_KEY="Optional, Production only"
+# For SMTP configuration
+SMTP_ADDRESS=
+SMTP_PORT=
+SMTP_DOMAIN=
+SMTP_USER_NAME=
+SMTP_PASSWORD=
+SMTP_AUTHENTICATION=
```

Comment:
we should probably document these in the installation guide now that we're making Sendgrid optional, we should also reflect that

Reply:
Good call! Added!